src/gui/vector/qgsdiagramproperties.h

```diff
@@ -203,6 +203,18 @@ class GUI_EXPORT QgsDiagramProperties : public QgsPanelWidget, private Ui::QgsDi
      */
     void insertDefaults();
 
+    /**
+     * Sets widgets to reflect the \a enabled status of the diagram.
+     * @param enabled Whether the diagram is enabled or not.
```

Comment:
```suggestion
     * Sets widgets to reflect the \a enabled status of the diagram.
     * \param enabled Whether the diagram is enabled or not.
     * 
     * \see enabledDiagram()
     * \since QGIS 3.40
```

Reply:
Done, thanks for the review.